### DISCUSSION TITLE
looking forward to PixArt - Ω

### DISCUSSION THREAD

**Discussion by YANG199301** [PixArt-alpha/PixArt-alpha#92] [General]
Hi,Thank you for your great project.
I am very curious and looking forward to PixArt - Ω. 
Can you introduce the differences and advantages of PixArt - Ω project in advance? 
Approximately when will it be released?

**Comment by lawrence-cj** [comment]
Thx for your noticing. The PixArt-Sigma is released: https://github.com/PixArt-alpha/PixArt-sigma
